pkg/cmd/workspace/util/branch_wizard.go

```diff
@@ -66,8 +66,9 @@ func GetBranchFromWizard(config BranchWizardConfig) (*apiclient.GitRepository, e
 	}
 
 	var branch *apiclient.GitBranch
+	parentIdentifier := config.ProviderId + "/" + config.NamespaceId + "/" + config.ChosenRepo.Name
```

Comment:
![Screenshot 2024-08-22 at 13 35 34](https://github.com/user-attachments/assets/9cf3fd49-e3ff-4d38-9d02-3eace35954ad)

Please use the namespace name instead of the ID to avoid the problem above (you can add the namespace name to the view config)

Reply:
@abhishek818 still getting `<PERSONAL>` after the latest changes

Reply:
yeah, i just realized a mistake and was about to push changes, 
Done now.

Reply:
Are you sure you're testing the changes? 😅  I got `<PERSONAL>` again.

![Screenshot 2024-08-22 at 17 11 33](https://github.com/user-attachments/assets/8d3cec2d-473b-4383-898f-5ff63ab86b8f)

Also, now I get `github//project-config-name` when creating a workspace from a project config.

Reply:
![2024-08-2221-02-57-ezgif com-video-to-gif-converter(1)](https://github.com/user-attachments/assets/9f2ba822-be4d-44f3-a799-cae0eb32016c)

Reply:
@Tpuljak

Reply:
Uploading 2024-08-22 21-02-57.mp4…

Reply:
https://github.com/user-attachments/assets/d60ce49e-f54c-4b2f-893c-381ffbd8fe8a

Reply:
![Screenshot 2024-08-22 at 17 49 44](https://github.com/user-attachments/assets/9321d4fe-9e71-4c63-81ac-ea50b800b2b3)

@abhishek818 it works for you because you only have one namespace. You need to set the namespace to the name in the `else` as well.

Reply:
Also, please address `dtn pc new && dtn create`, then choose a project config

Reply:
done..